subscriber/common/job/controller.go

```diff
@@ -150,6 +166,51 @@ func NewController(params Params) *Controller {
 	return controller
 }
 
+func connectEtcdServices(params Params) (services.Services, error) {
+	iopts := instrument.NewOptions().
+		SetZapLogger(params.ServiceConfig.Logger).
+		SetMetricsScope(params.ServiceConfig.Scope)
+
+	// etcd key format: prefix/${env}/namespace/service/instanceId
+	params.ServiceConfig.EtcdConfig.Service = fmt.Sprintf("%s/%s",
+		config.ActiveAresNameSpace, params.ServiceConfig.EtcdConfig.Service)
```

Comment:
any typo? or reassign?

Reply:
reassign EtcdConfig.Service which is defined in m3db